rust-keybroker/keybroker-server/src/main.rs

```diff
@@ -99,6 +99,14 @@ async fn submit_evidence(
 
     let evidence_bytes = URL_SAFE_NO_PAD.decode(evidence_base64).unwrap(); // TODO: Error handling needed here in case of faulty base64 input
 
+    // Optionally dump the evidence to file.
+    // This can be useful for debugging or for educational purpose for example.
+    if data.args.dump_evidence_cbor {
+        let filename = format!("evidence-{challenge_id}.cbor",);
+        std::fs::write(&filename, &evidence_bytes).unwrap();
```

Comment:
exploding on this kind of error seems a bit drastic, no? :)

Reply:
Ok, I'll add an IO error kind then. This might get useful sometime in the future.